agent/src/main/resources/build-scan-init.gradle

```diff
@@ -1,22 +1,15 @@
 import org.gradle.util.GradleVersion
 
-// finish early if the build is an included build of a composite build
-def isTopLevelBuild = !gradle.parent
-if (!isTopLevelBuild) {
-    return
-}
```

Comment:
I would have left this at the top, even if it means we do this check very early and the next check further down.

Reply:
OK. I think having this block above `initscript` makes it appear as though this check will prevent the `initscript` block from having an effect. This is not the case, since the `initscript` block is extracted and processed separately from the rest of the build.

That's why I moved the check down.

Reply:
Thanks for the insights. We now have two checks: one in the main init script and one in the `initscript` block.